cmd/node/main.go

```diff
@@ -346,6 +354,8 @@ func main() {
 	_ = logger.SetDisplayByteSlice(logger.ToHexShort)
 	log := logger.GetOrCreate("main")
 
+	time.Sleep(2 * time.Second)
```

Comment:
why is this needed here ? add a constant.

Reply:
fixed. moved to the startInEpoch flag verification.